dir.props

```diff
@@ -72,7 +72,7 @@
       <ExpectedVersion>2.1.0</ExpectedVersion>
     </ValidationPattern>
     <ValidationPattern Include="^(?i)(NuGet\..%2A)$">
-      <ExpectedVersion>3.5.0-beta2-1456</ExpectedVersion>
+      <ExpectedVersion>3.5.0-rc-1590</ExpectedVersion>
```

Comment:
Yes, it is a version that has this change.  For pre-release builds of NuGet we don't often have a good LKG so I pick latest and then test it and use it if its good.  I've mirrored this to the buildtools feed.